src/dparse/parser.d

```diff
@@ -7374,6 +7379,16 @@ protected: final:
         if (!moreTokens()) return false;
         switch (current.type)
         {
+        case tok!("__traits"):
```

Comment:
This new block causes the bad coverage report because `isDeclaration` is surprisingly never used by the parser.
However i've updated it because it can be used by 3rd parts applications.